### PR TITLE
Feature/dss 6vs7

### LINKED ISSUE #166: HecLib 6 and 7 don't work on the same thread together
This was originally reported by @hxiedwr in #163. 

HecLib 6 and 7 don't work on the same thread together, i.e. opening dvar (heclib 7) and svar (heclib 6) together on DSS perspective.

### PR DESCRIPTION
**Pull request by rma-tom**
Branch and Pull Request in Response to issue #166

### CODE REVIEW

**Review by jmdegeorge — APPROVED** [review]
Updated DSS path matching configuration looks good to me.

**Comment by rma-tom** [comment]
Reviewed and approved changes to DataOps.java and DSSCatalogView.java to resolve display of both DSS6 and DSS7 contents in schematic

**Comment by jmdegeorge** [comment]
Added fix to resolve Schematics not loading values for DSS6 data sets. Forced dss path keys to lower case to match lower case expectations.

**Review by jmdegeorge — APPROVED** [review]
Still good after rebase.

**Review by jmdegeorge — DISMISSED** [review]
Looks good!

File: dwr-hecdssvue/src/main/java/gov/ca/dwr/hecdssvue/components/DataOps.java [hunk 1]
```diff
@@ -423,7 +423,8 @@ public static HashMap<String, String> generatePathnameMap(HecDss file, int j){
 			String pathname=cr.getNominalPathname();
 			DSSPathname dssPathname = new DSSPathname(pathname);
 			String partB=dssPathname.bPart();
-			pathnameMap.put(partB, pathname);
+			// locase for case insensitive search
```
Comment (jmdegeorge) [inline dwr-hecdssvue/src/main/java/gov/ca/dwr/hecdssvue/components/DataOps.java]: Please fix to say "lowercase"

**Review by jmdegeorge — APPROVED** [review]
Thanks for the clarification.

**Pull request merged**